StPicoEvent/StPicoEvent.h

```diff
@@ -110,6 +111,8 @@ class StPicoEvent : public TObject
   UShort_t mFillId;
   Float_t  mBField;
 
+  Int_t    mTime;//To set timestamp for St_db_Maker. This is what StMuDstMaker used to GetEvtHddr()->SetGMTime(cast into unsigned int). Peifeng Liu
```

Comment:
Could you prepare a small example to demonstrate this? I am not surprised that St_db_Maker complains when it does not have any reference to the time but it is surprising that adding the time to StPicoEvent solves this problem.

I also don't see how St_db_Maker would pick up the right time in this macro https://github.com/star-bnl/star-picoDst/blob/master/makePicoDst.C
Perhaps, it is a good question for the picoDst-hn mailing list and those to wrote the macro.